tests/js/client/shell/shell-index.js

```diff
@@ -0,0 +1,109 @@
+/*jshint globalstrict:false, strict:false */
+/*global fail, assertEqual, assertNotEqual, assertTrue, assertFalse, assertNull */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test index methods
+///
+/// DISCLAIMER
+///
+/// Copyright 2024 ArangoDB GmbH
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is triAGENS GmbH, Cologne, Germany
+///
+/// @author Kaveh Vahedipour
+////////////////////////////////////////////////////////////////////////////////
+
+const jsunity = require("jsunity");
+const internal = require("internal");
+const arango = internal.arango;
+const errors = internal.errors;
+const { helper, versionHas } = require("@arangodb/test-helper");
+const platform = require('internal').platform;
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test suite: basics
+////////////////////////////////////////////////////////////////////////////////
+
+function IndexSuite() {
+  'use strict';
+
+  return {
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test: indexes
+////////////////////////////////////////////////////////////////////////////////
+
+    testIndexProgress : function () {
+      let c = internal.db._create("c0l", {numberOfShards: 3, replicationFactor: 2});
```

Comment:
It looks like the changes to the test are missing in the forward-port.

Reply:
rechecking.

Reply:
that is weird. the diff between the tests in 3.10/3.11 is empty.

Reply:
fixed